kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/DhcpConfigReader.java

```diff
@@ -241,20 +234,10 @@ private DhcpServerConfig4 populateDhcpdConfig(String interfaceName, File dhcpCon
                         rangeEnd, dnsList);
                 dhcpServerConfigIP4 = new DhcpServerConfigIP4(dhcpServerCfg, dhcpServerCfgIP4);
             } catch (KuraException e) {
-                s_logger.error("Failed to craete new DhcpServerConfigIP4 object - {}", e);
+                logger.error("Failed to create new DhcpServerConfigIP4 object - {}", e);
             }
-        } catch (FileNotFoundException e) {
-            throw new KuraException(KuraErrorCode.CONFIGURATION_ERROR, e);
         } catch (IOException e) {
             throw new KuraException(KuraErrorCode.CONFIGURATION_ERROR, e);
-        } finally {
```

Comment:
It's been refactored to use the new (as of Java 7) try-with-resources. It will close the (Auto)Closeable in the end:
```java
try (BufferedReader br = new BufferedReader(new FileReader(dhcpConfigFile))) {
    // work with br, here
}
// br is safely closed at this point
```

But to be honest, if an `IOException` occurs closing the stream, it will no longer be logged like so:
```java
s_logger.error("I/O Exception while closing BufferedReader!");
```
The `IOException` would be caught by the catch statement above and `KuraException` would be thrown instead of the exception being logged and suppressed. That's the reason I stopped refactoring such cases in some of the later PRs.
So, if you suggest to keep the existing behavior where exception during `close()` is logged and suppressed, I will undo the changes.